### ISSUE TITLE
JAWS is not announcing the input type properly

### ISSUE THREAD

**Issue by vaishaliganesan** [ckeditor/ckeditor5#12695]
After pressing enter key for the link, the focus would be redirected to input text where I have added the aria-label as "Verification code resend was successful"
### ✔️ Expected result
JAWS should announce as "Verification code resend was successful Edit"

### ❌ Actual result
But It is announcing as "Verification code resend was successful link" which is strange and it is happening only in chrome and edge browsers.

Note - NVDA is working as expected.

* Browser: …
Chrome and Edge
Working as expected in firefox

---

Code Snippet

Link id -  #resentOtp
Input text id - #OTP   
$("#resentOtp").click(function (e) {
            $.ajax({
                url: "@Url.Action("MFAResendOtp", "Account")",
                type: "Post",
                async: false,
                data: { type: email/Phone,
                success: function (response) {
                    if (response) {
                        closeWaitingDialog();
                        $("#resendOtpSucc").show();
                        $("#OtpVerifyFailed").hide();
                        $("#OTP").val("").removeClass("input-validation-error");
                        $("#OTPValidation").remove();
                        $("#resentOtp").prop("disabled", false);
                        $("#OTP").focus();
                        $("#OTP").attr('aria-label', 'Verification code resend was successful');
                        setTimeout(function () {
                            $("#resendOtpSucc").hide();
                            $("#OTP").removeAttr('aria-label');
                        }, 5000);
                    }
                    else {
                        $("#OTP").removeAttr('aria-label');
                    }
                  }
            }); 

Kindly let me know if you have any idea

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CKEditorBot** [comment]
We've closed your issue due to inactivity over the last year. We understand that the issue may still be relevant. If so, feel free to open a new one (and link this issue to it).
